samples/EdgeConfig/shared-pom.xml

```diff
@@ -98,6 +98,13 @@
                             <goal>flowhooks</goal>
                         </goals>
                     </execution>
+                    <execution> 
+                        <id>create-config-customroles</id>
+                        <phase>install</phase>
+                        <goals>
+                            <goal>customroles</goal>
```

Comment:
Can we call it `userroles` in stead of `customroles` matching the name of the resource in our Management API.